### PR TITLE
Remove the old CI builder and update IMP weights

### LINKED ISSUE #1765: Follow-up of #1761: re-benchmark pallet-IMP
### Context

As topic, one of the follow-up TODOs of https://github.com/litentry/litentry-parachain/pull/1761

We need to re-benchmark pallet-IMP due to extrinsic changes.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### LINKED ISSUE #1778: Remove `dev-builder` in GHA
### Context

The server was used to compile the code and build docker images mainly.

Now that we removed it from the runner list, we need to make adjustments in GHAs accordingly (= replace it with github runner)

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by Kailai-Wang**
resolves #1778 
resolves #1765 

I had some problems building images with `runtime-benchmarks` feature, which was later solved by increasing the swap size for the cloud instance.

### CODE REVIEW

File: .github/workflows/build-docker-with-args.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ env:
 jobs:
   ## build docker image of client binary with args ##
   build-docker-with-args:
-    runs-on: dev-builder
+    runs-on: ubuntu-latest
```
Comment (BillyWooo) [inline .github/workflows/build-docker-with-args.yml:30]: why use `ubuntu-latest` here, but `ubuntu-22.04` above?

File: .github/workflows/create-release-draft.yml [hunk 2]
```diff
@@ -104,7 +104,7 @@ jobs:
   ## build docker image of client binary ##
   build-docker:
     if: ${{ github.event.inputs.release_client == 'true' }}
-    runs-on: dev-builder
+    runs-on: ubuntu-latest
```
Comment (BillyWooo) [inline .github/workflows/create-release-draft.yml:107]: same here.

**Review by BillyWooo — APPROVED** [review]
LGTM

File: .github/workflows/build-docker-with-args.yml (hunk 1 above)
Comment (Kailai-Wang) [reply .github/workflows/build-docker-with-args.yml:30]: they are synonyms basically, let me change them all to `ubuntu-latest`

**Pull request merged**